src/project/qml/MuseScore/Project/internal/GeneralInfoView.qml

```diff
@@ -81,8 +81,6 @@ Column {
 
             title: qsTrc("project", "Composer")
 
-            info: qsTrc("project", "Composer / arranger")
```

Comment:
You shouldn't remove this value (the same for the subtitle field), it should be used for all scores by default